client/lib/abtest/active-tests.js

```diff
@@ -113,5 +113,13 @@ module.exports = {
 			hideVerticalScreenshots: 50,
 		},
 		defaultVariation: 'hideVerticalScreenshots',
+	},
+	siteTitleStep: {
+		datestamp: '20160928',
+		variations: {
+			showSiteTitleStep: 95,
+			hideSiteTitleStep: 5,
```

Comment:
These are reversed only until the code gets ready to merge, so we can test the variation and adding of step.

I will revert that before merging.